File: 15-hacktiv/6-check-vowel/solution.js
function checkAllVowels(str) {
  let vowel = ['a', 'i', 'u', 'e', 'o']
  for (let s of str) {
    c = s.toLowerCase()
    if (vowel.indexOf(c) === -1) {
      return false
    }
  }
  return true
}

console.log(checkAllVowels('AiUeO')) // true
console.log(checkAllVowels('aioeo')) // true
console.log(checkAllVowels('AIUEO')) // true
console.log(checkAllVowels('NotAllVowels')) // false

File: 15-hacktiv/6-check-vowel/solution.py
def checkAllVowels(str):
  vowel = ['a', 'i', 'u', 'e', 'o']
  for s in str:
    c = s.lower()
    if c not in vowel:
      return False
  return True

print(checkAllVowels('AiUeO')) # true
print(checkAllVowels('aioeo')) # true
print(checkAllVowels('AIUEO')) # true
print(checkAllVowels('NotAllVowels')) # false